Feature/sunmpicomm (#370)
LLNL/sundials: doc/shared/sundials/Install.rst

table names are relative to ``LIBDIR`` for libraries and to ``INCLUDEDIR`` for
header files.

Using SUNDIALS in your prpject
------------------------------

> After building and installing SUNDIALS, you probably want to include SUNDIALS in
> your application. This involves two steps: including the right header files, and
> linking to the right libraries.

Author: ```suggestion
After building and installing SUNDIALS, using SUNDIALS in your application involves
two steps: including the right header files and linking to the right libraries.
```
